codecov.yaml

```diff
@@ -0,0 +1,24 @@
+codecov:
+  token: a2b1c854-8627-4747-89cb-c312da33227d
+  require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "0...100"
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
```

Comment:
curious about why this is needed for Golang coverage?

Reply:
Not be needed, I copied from the default template.
Removed.